samples/browser/src/main/java/org/mozilla/samples/browser/DefaultComponents.kt

```diff
@@ -16,28 +16,18 @@ import mozilla.components.browser.search.SearchEngineManager
 import mozilla.components.browser.session.Session
 import mozilla.components.browser.session.SessionManager
 import mozilla.components.browser.session.storage.DefaultSessionStorage
-import mozilla.components.concept.engine.DefaultSettings
 import mozilla.components.concept.engine.Engine
 import mozilla.components.feature.search.SearchUseCases
 import mozilla.components.feature.session.SessionIntentProcessor
 import mozilla.components.feature.session.SessionUseCases
 import mozilla.components.feature.session.TextSearchHandler
 import mozilla.components.feature.tabs.TabsUseCases
-import org.mozilla.geckoview.GeckoRuntime
-import org.mozilla.samples.browser.request.SampleRequestInterceptor
 
-/**
- * Helper class for lazily instantiating components needed by the application.
- */
-class Components(private val applicationContext: Context) {
-    // Engine
-    val engine: Engine by lazy {
-        val defaultSettings = DefaultSettings().apply {
-            requestInterceptor = SampleRequestInterceptor()
-        }
+open class DefaultComponents(private val applicationContext: Context) {
 
-        val runtime = GeckoRuntime.getDefault(applicationContext)
-        GeckoEngine(runtime, defaultSettings)
+    // Engine
+   open val engine: Engine by lazy {
```

Comment:
This formatting looks a bit off. I see 3 space indentation here but 4 is others.

I haven't written the docs for this yet, but we should be using `ktlint --apply-to-idea-project --android` and that takes care of the formatting issues.

Reply:
You're right!
I just fixed it! 👍